src/components/modals/TransactionAccelerateTransactionModal.js

```diff
@@ -62,15 +62,19 @@ class TransactionAccelerateModalComponent extends PureComponent<Props, State> {
   makeRbfTransaction = async () => {
     const { edgeTransaction, wallet } = this.props
 
-    if (edgeTransaction.spendTargets) {
-      const spendTargets = edgeTransaction.spendTargets.map(spendTarget => ({
+    const edgeTransactionSpendTargets = edgeTransaction.spendTargets
+
+    if (edgeTransactionSpendTargets) {
+      const spendTargets = edgeTransactionSpendTargets.map(spendTarget => ({
```

Comment:
Check if `spendTargets` is void or empty. Under this condition, set error state. Also do preliminary check on spendTargets in parent component for `isAcceleratable`.